src/api/server.tsx

```diff
@@ -110,6 +110,33 @@ export const fetchResources = async (
     });
 };
 
+export const fetchClusterMembersResources = async (
+  clusterMembers: LxdClusterMember[],
+): Promise<LxdResources[]> => {
+  return new Promise((resolve, reject) => {
+    Promise.allSettled(
+      clusterMembers.map(async (member) => {
+        return fetchResources(member.server_name);
+      }),
+    )
+      .then((results) => {
+        const data: LxdResources[] = [];
+        for (let i = 0; i < clusterMembers.length; i++) {
+          const member = clusterMembers[i];
+          const result = results[i];
+          if (result.status === "fulfilled") {
+            const promise = result;
+            data.push(promise.value);
```

Comment:
```suggestion
            data.push(result.value);
```
Why the extra variable?